### ISSUE TITLE
Rebuild image

### ISSUE THREAD

**Issue by dereuromark** [alt3/cakebox-builder#6]
We currently have no image uploaded

In order to rebuild, we need to get this tool first running again
The docs could maybe help by also outlining quickly how to install
- Chef-DK
- Berksfile
- Test-kitchen

or a link to those resources.

**Comment by bravo-kernel** [comment]
The first two will pop up as first hit on Google. The last one if you google "chef test-kitchen".

It's not really complex once you get all the tooling installed locally. All scripts required are in this repo.

Shout out if you get stuck ok?

**Comment by dereuromark** [comment]
Hah, I tried locally, and I failed misarably with every single of of them
Linux Mint
I am no dev op, and I will never be... :)

**Comment by bravo-kernel** [comment]
Yeah, I understand and am really sorry about that man. If there was any chance I would give this a go myself but I simply do not have the time. Can't believe they hard-deleted the files 🤦
